app/src/main/java/io/left/meshim/adapters/MessageListAdapter.java

```diff
@@ -124,6 +134,19 @@ public void onBindViewHolder(MessageViewHolder holder, int position) {
                 if (sender != null) {
                     holder.mUserImage.setImageResource(sender.getAvatar());
                 }
+                if(message.getFilePath()!= "") {
```

Comment:
nitpicky but there should be spaces after the "if"

Reply:
And before the `!=`. Would be worth running Checkstyle over the codebase.